pulseapi/tests.py

```diff
@@ -82,6 +86,22 @@ def setup_users_with_profiles(test):
     test.users_with_profiles = users
 
 
+class JSONDefaultClient(Client):
+    """
+    Same as a regular test client except the default content type is 'application/json'
+    for the post method instead of 'multipart/form-data'
+    """
+    def post(self, path, data=None, content_type=CONTENT_TYPE_JSON,
+             follow=False, secure=False, **extra):
+        return super(JSONDefaultClient, self).post(
```

Comment:
it looks like `follow` gets lost in this super call.

Reply:
good catch!